packages/dapp/src/utils/constants/twitter.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  shareIntent: 'https://twitter.com/intent/tweet?text=',
+  shareContent: 'Checkout my collection! <COLLECTION_LINK>' // @todo!
```

Comment:
shareContent: 'Come and make eternally beautiful marks with me at https://sign.kernel.community so we can reimagine what significant money actually means.'

Make sure to include a shareImage, which is the NFT itself, as that is also important for Twitter purposes.

Reply:
that would come from the SEO no?